libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -428,6 +428,29 @@ void AP_BattMonitor::check_failsafes(void)
     }
 }
 
+// returns the total power draw for all batteries
+float AP_BattMonitor::power_watts() const
```

Comment:
summing causes issues with some battery types (fuel systems, already summed batteries etc)
better to add a OSD_BAT_IDX param